crates/tests-e2e/tests/send_payment_when_open_position.rs

```diff
@@ -10,7 +10,7 @@ async fn can_send_payment_with_open_position() {
     let app = &test.app;
 
     let ln_balance_before = app.rx.wallet_info().unwrap().balances.lightning;
-    let invoice_amount = 10_000;
+    let invoice_amount = 1_000;
```

Comment:
🤔 This change doesn't seem to belong in this patch.

Reply:
yup, thanks for spotting it! got accidentally tagged along after our last session